gc/base/GCExtensionsBase.hpp

```diff
@@ -906,6 +906,9 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 	MMINLINE OMR_VM* getOmrVM() { return _omrVM; }
 	MMINLINE void setOmrVM(OMR_VM* omrVM) { _omrVM = omrVM; }
 
+#if defined(OMR_GC_MODRON_SCAVENGER)
+	virtual void registerScavenger(MM_Scavenger *scavenger) { this->scavenger = scavenger; }
```

Comment:
for symmetry, let's introduce unregister, too

i'd be used in MM_ConfigurationGenerational::tearDown

Reply:
See https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#variables.

In C++, member variable names should start with an underscore: `_scavenger`. This is just a note and does not need to be addressed in this PR since the `scavenger` member field was introduced in the past.

Reply:
Unfortunately whole that class/file does not follow that standard, although by far most of other GC classes do. 

Being a top level global GC class, conforming it to the standard would likely touch almost every GC file.

That's probably why nobody so far have tried to do it, but it should have really be done during the major OMR/J9 separation.